README.md

```diff
@@ -69,5 +70,70 @@ Usage:
 ./add_or_update.py [community.json OR official.json] [github/gitlab url OR app name [github/gitlab url OR app name [github/gitlab url OR app name ...]]]
 ```
 
+### How to make my app a High Quality app ?
+
+A High Quality app will be highlighted in the app list and marked as a level 8 app.  
+To become a High Quality app, a package has to follow the following rules:
+
+* The app should already have been in the community list for 2 months.
+* The app should be kept up to date, regarding the upstream source (if it’s possible with our current YunoHost version).
+* The package itself should be up to date regarding the packaging recommendations and helpers.
+* The package should be level 7, at least.
```

Comment:
Shouldn't we also add the requirement for X months / weeks here ?

Reply:
Yep I guess